tests/integration/home-page/home-page.spec.ts

```diff
@@ -244,3 +101,82 @@ describe('Home page tests on desktop', () => {
     })
   })
 })
+
+const isSearchBarValueIn = (value: string | undefined, searchBarValue: string) => {
+  return value?.toLowerCase().includes(searchBarValue.toLowerCase())
+}
+
+const isSearchBarValueInTag = (tags: string[] | undefined, searchBarValue: string) => {
+  return tags?.some((tag) => (tag.toLowerCase().includes(searchBarValue.toLowerCase())))
+}
+
+const filterDevfilesOnSearchBar = (devfiles: Devfile[], searchBarValue: string): Devfile[] => {
+  if (searchBarValue === '') {
+    return devfiles
+  }
+
+  const devfilesFilteredOnSearchBar: Devfile[] = devfiles.filter((devfile: Devfile) => {
```

Comment:
pls add testing partitions and/or what UI is being tested  in general throughout file

Reply:
This is not necessary as long as the test description is good enough